run.go

```diff
@@ -86,6 +95,8 @@ func RunCmdInteractively(ctx context.Context, rootCmd *cobra.Command, cnfg *haze
 	})
 	flagsToExclude = append(flagsToExclude, "help")
 	p.FlagsToExclude = flagsToExclude
+	rootCmd.Example = fmt.Sprintf("> %s\n> %s", mapcmd.MapPutExample, mapcmd.MapGetExample) + "\n> cluster version"
+	rootCmd.Use = ""
```

Comment:
What do these do? I did not get it.

Reply:
These are to modify the root command for interactive mode. We want to see different example text on help command on  interactive mode.

Reply:
It is now seen as:

```
Usage:
   [flags]
   [command]
```

Is it something we want ? Can't we delete `Usage` completely?

Reply:
Let's not extend the scope of this PR. I created an issue for this https://github.com/hazelcast/hazelcast-commandline-client/issues/68